conf/documentation.conf

```diff
@@ -398,6 +363,13 @@ description=<<EOT
 Should routes be managed by PacketFence?
 EOT
 
+[services.netdata]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Should netdata be managed by PacketFence?
+EOT
+
```

Comment:
Seems to be missing the new collectdtostatsd service configuration in this file

Reply:
done